src/System.Management.Automation/resources/Modules.resx

```diff
@@ -141,15 +141,6 @@
   <data name="ManifestMemberNotFound" xml:space="preserve">
     <value>The module to process '{0}', listed in field '{1}' of module manifest '{2}' was not processed because no valid module was found in any module directory.</value>
   </data>
-  <data name="ScriptsFileNotFound" xml:space="preserve">
```

Comment:
Seems odd that some of this is removed when it's not workflow specific?

Reply:
Yes, it is not related workflow but it is "dead" code so I clean up it too. It seems it make no sense to do this in another PR.

Reply:
Before the fix:
```
ipmo : Cannot load the workflow module 'XamlTest.xaml'. Workflow is not supported in PowerShell 6+.
At line:1 char:1
+ ipmo C:\temp\1\XamlRootModule.psd1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Import-Module], NotSupportedException
+ FullyQualifiedErrorId : Modules_WorkflowModuleNotSupported,Microsoft.PowerShell.Commands.ImportModuleCommand
```

After the fix (there was a bug - fixed in new commit):
```
ipmo : The module to process 'XamlTest.xaml', listed in field 'NestedModules' of module manifest '' was not processed. The extension '.xaml' is not a valid module extension. The supported module extensions are '.dll', '.ps1', '.psm1', '.psd1' and '.cdxml'. Correct the extension then try adding the file 'XamlTest.xaml' again.
At line:1 char:1
+ ipmo C:\temp\1\XamlRootModule.psd1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Import-Module], PSInvalidOperationException
+ FullyQualifiedErrorId : Modules_InvalidModuleExtension,Microsoft.PowerShell.Commands.ImportModuleCommand
```